nucypher/characters/base.py

```diff
@@ -67,6 +68,7 @@ class Character(Learner):
     from nucypher.crypto.signing import InvalidSignature
 
     def __init__(self,
+                 domains: Set = (constants.GLOBAL_DOMAIN,),
```

Comment:
Well - which is it, a set or a tuple? :-)

Reply:
`NamedEmptySet`

Reply:
Actually, kidding aside, what do you think of this?  It is OK to presume that a Set - and only a Set - is passed, but that a tuple is default?

I was just going to change it to a set when I realized that it actually makes more sense for the default to be immutable.

Reply:
I'm inclined to remove this from the signature entirely and use `None`.